milli/src/update/index_documents/extract/extract_fid_docid_facet_values.rs

```diff
@@ -46,15 +54,19 @@ pub fn extract_fid_docid_facet_values<R: io::Read + io::Seek>(
 
         for (field_id, field_bytes) in obkv.iter() {
             if faceted_fields.contains(&field_id) {
-                let value =
-                    serde_json::from_slice(field_bytes).map_err(InternalError::SerdeJson)?;
-                let (numbers, strings) = extract_facet_values(&value);
-
                 key_buffer.clear();
 
+                // here, we know already that the document must be added to the “field id exists” database
                 // prefix key with the field_id and the document_id
+
                 key_buffer.extend_from_slice(&field_id.to_be_bytes());
                 key_buffer.extend_from_slice(&docid_bytes);
+                fid_docid_facet_exists_sorter.insert(&key_buffer, ().as_bytes())?;
```

Comment:
I think we should benchmark it, but, shouldn't it be faster if we directly push the `field_id` as the key and the `docid` as the value replacing the `keep_first` merger with a `merge_cbo_roaring_bitmaps`?

Reply:
I had some problems when I tried to do that at first because, but I tried again this morning and it seems to work :-)